tsfresh/utilities/string_manipulation.py

```diff
@@ -48,10 +48,19 @@ def convert_to_output_format(param):
 
        <param name>_<param value>__<param name>_<param value>__ ...
 
+    If a <param_value> is a string, this method will wrap it with parenthesis ", so "<param_value>"
+
     :param param: The dictionary of parameters to write out
     :type param: dict
 
     :return: The string of parsed parameters
     :rtype: str
     """
-    return "__".join(str(key) + "_" + str(param[key]) for key in sorted(param))
+
+    def add_paranthesis_if_string_value(x):
+        if isinstance(x, numbers.Number):
```

Comment:
Don't you want to turn this the other way round: only add " on str (or basestring)? Because we may also have lists and tuples and stuff, where we do not want to have parenthesis.

Also, it is par*e*nthesis, not par*a*nthesis ;-)